docs/source/install.md

```diff
@@ -0,0 +1,85 @@
+# Install Fabric and Fabric Samples
+
+Please install the [Prerequisites](./prereqs.html) before following these install instructions.
+
+We think the best way to understand something is to use it yourself.  To help you use Fabric, we have created a simple Fabric Network called `test-network` using docker compose, and a set of sample applications that demonstrate its core capabilities.
+We have also precompiled `Fabric CLI tool binaries` and `Fabric Docker Images` which will be downloaded to your environment, to get you going.
+
+The cURL command in the instructions below sets up your environment so that you can run the 'test-network'. Specifically, it performs the following steps:
+* Clones the [hyperledger/fabric-samples](https://github.com/hyperledger/fabric-samples) repository.
+* Downloads the latest Hyperledger Fabric docker images and tags them as `latest`
+* Downloads the following platform-specific Hyperledger Fabric CLI tool binaries and config files into the fabric-samples `/bin` and `/config` directories. These binaries will help you interact with the 'test-network'.
+  * `configtxgen`,<br>
+  * `configtxlator`,<br>
+  * `cryptogen`,<br>
+  * `discover`,<br>
+  * `idemixgen`,<br>
+  * `orderer`,<br>
+  * `osnadmin`,<br>
+  * `peer`,<br>
+  * `fabric-ca-client`,<br>
+  * `fabric-ca-server`<br>
+
+
+## Download Fabric samples, docker images, and binaries.
+
+Download fabric-samples to the `$HOME/go/src/github.com/<your_github_userid>` directory.  This is a Golang Community recommendation for Go projects. If you are using a different directory or Windows, see the [Notes](https://hyperledger-fabric.readthedocs.io/en/latest/install.html#notes) below.
+
+```shell
+$ mkdir -p $HOME/go/src/github.com/<your_github_userid>
+$ cd $HOME/go/src/github.com/<your_github_userid>
+```
+
+Download the latest release of Fabric samples, docker images, and binaries.
+
+```shell
+$ curl -sSL https://bit.ly/2ysbOFE | bash -s
+```
+
+You have completed installing Fabric samples, docker images, and binaries to your system.
+
+## Advanced download options
+
+To view the help and available commands for the download script, please use the `-h` flag with the cURL command:
+
+```shell
+curl -sSL https://bit.ly/2ysbOFE | bash -s -- -h
+```
+
+To download a specific release, pass a version identifier for Fabric and Fabric-CA docker images. The command below demonstrates how to download the latest production releases - `Fabric v2.3.1` and `Fabric CA v1.4.9` 
```

Comment:
The Fabric CA generally calls itself "Fabric CA", not "Fabric-CA: https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/ca-deploy.html

Also "Docker" again here.

Reply:
done.